File: atividade_cpp/r/q2.cpp
#include <iostream>

using namespace std;

float somatorio(float v[5]){
    float soma = 0;

    for(int i=0; i < 5; i++){
        soma = soma + v[i];
    }
    return soma;
}

int main(){
    float v[5];
    float n = 0, soma = 0;

    for(int i=0; i < 5 ; i++){
        cout << "Digite um valor: ";
        cin >> n;
        v[i] = n; 
    }

    soma = somatorio(v);
    cout << "Somatório do vetor: " << soma << endl;

    return 0;
}

File: JAVASCRIPT/REVISAO-02/q2.js
//Crie uma função que receba um vetor de números como parâmetro e retorne a soma de todos os elementos do vetor
let vetor = [];
let valor = 0;
for (let i = 0; i < 3; i++) {
  valor = parseInt(prompt("Digite um valor: "));
  vetor.push(valor);
}

function vetorSoma(x) {
  let valores = 0;
  for (i = 0; i < x.length; i++) {
    valores = valores + x[i];
  }
  console.log(`A soma dos valores é: ${valores}`);
}

vetorSoma(vetor);